hail/src/main/scala/is/hail/annotations/BroadcastValue.scala

```diff
@@ -22,7 +22,7 @@ case class SerializableRegionValue(encodedValue: Array[Byte], t: PType, makeDeco
 }
 
 object BroadcastRow {
-  def empty(ctx: ExecuteContext): BroadcastRow = apply(ctx, Row(), TStruct())
+  def empty(ctx: ExecuteContext): BroadcastRow = apply(ctx, Row(), TStruct.empty)
```

Comment:
for consistency lets either choose no parens or parens

Reply:
TStruct.empty is a val, not a ef

Reply:
also, notice the `def apply(): this.type = this` everywhere on the case object tpyes (TInt32 for example). This will get removed in a followup PR, didn't want a more complicated diff than necessary.

Reply:
ah I see